sphinx_book_theme/__init__.py

```diff
@@ -386,3 +386,5 @@ def setup(app: Sphinx):
     app.connect("html-page-context", add_to_context)
 
     app.add_directive("margin", Margin)
+
+    return {"parallel_read_safe": True}
```

Comment:
```suggestion
    return {"parallel_read_safe": True, "parallel_write_safe": True}
```

While we are at it. pydata-sphinx-theme (which this derives from) sets both of these also, so again I don't believe it should be an issue